src/components/Search.js

```diff
@@ -132,7 +136,7 @@ class SearchModal extends React.Component {
     db.get(AutographaStore.bookId.toString()).then((doc) => {
         if (AutographaStore.replaceOption === "chapter") {
             for (var c in replacedChapter) {
-                var verses = doc.chapters[AutographaStore.chapterId-1].verses
+                var verses = (doc.chapters[AutographaStore.chapterId-1]).verses
```

Comment:
Was this the logical error you were talking about?

Reply:
nope this is just auto-formatting done by prettier

Reply:
@joelthe1 I have changed loader text to gif